packages/backend-dynamic-feature-service/src/scanner/plugin-scanner.ts

```diff
@@ -37,6 +36,15 @@ export interface ScanRootResponse {
 
 export const configKey = 'dynamicPlugins';
 
+class WrappedError extends Error {
```

Comment:
could prolly use ForwardedError from the errors package instead

Reply:
Fixed